packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -282,7 +282,6 @@ const PostsPagePostHeader = ({post, answers = [], dialogueResponses = [], toggle
           {isEAForum ? <LWTooltip title={postGetCommentCountStr(post, commentCount)}>
             {commentCountNode}
           </LWTooltip> : commentCountNode}
-          {isEAForum && <BookmarkButton post={post} variant='iconWithText' />}
```

Comment:
Agree that this was not the intended behavior, and that we should be looking to remove the text instead.